src/main/java/spoon/support/adaption/NodePrintHelper.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-License-Identifier: (MIT OR CECILL-C)
+ *
+ * Copyright (C) 2006-2019 INRIA and contributors
+ *
+ * Spoon is available either under the terms of the MIT License (see LICENSE-MIT.txt) of the Cecill-C License (see LICENSE-CECILL-C.txt). You as the user are entitled to choose the terms under which to adopt Spoon.
+ */
+package spoon.support.adaption;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+final class NodePrintHelper {
```

Comment:
Add a short sentence with the information that this an utility class, and it's provided API.
As this is a package private final class, you don't need to write a lot of text.